.circleci/config.yml

```diff
@@ -101,13 +101,15 @@ jobs:
           name: Locale
           command: yarn run locale:verify
       - run:
-          name: Unit Tests
+          name: Run unit tests and publish to codeclimate
           command: |
-            yarn test:unit --coverage
-            if [ ! -z "$COVERALLS_REPO_TOKEN" ]; then
-              echo "Uploading coverage results to coveralls.io..."
-              cat ./coverage/lcov.info | npx coveralls
-            fi
+            curl -L https://codeclimate.com/downloads/test-reporter/test-reporter-latest-linux-amd64 > ./cc-test-reporter
```

Comment:
You should not download this without checking the checksum and signature. We had a security issue with this before. Other repos have stored this file in the system actually. Instead we can probably use this github action https://github.com/paambaati/codeclimate-action
@enell asked for permission from SSO and seems to be okay

Reply:
I will include the binary